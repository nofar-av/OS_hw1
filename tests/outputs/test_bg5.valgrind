==27001== Memcheck, a memory error detector
==27001== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27001== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27001== Command: /home/omer/OS_wet1/smash
==27001== Parent PID: 26999
==27001== 
==27001== 
==27001== FILE DESCRIPTORS: 4 open at exit.
==27001== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_bg5.valgrind
==27001==    <inherited from parent>
==27001== 
==27001== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_bg5.err
==27001==    <inherited from parent>
==27001== 
==27001== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_bg5.out
==27001==    <inherited from parent>
==27001== 
==27001== Open file descriptor 0:
==27001==    <inherited from parent>
==27001== 
==27001== 
==27001== HEAP SUMMARY:
==27001==     in use at exit: 0 bytes in 0 blocks
==27001==   total heap usage: 70 allocs, 70 frees, 84,672 bytes allocated
==27001== 
==27001== All heap blocks were freed -- no leaks are possible
==27001== 
==27001== For counts of detected and suppressed errors, rerun with: -v
==27001== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
