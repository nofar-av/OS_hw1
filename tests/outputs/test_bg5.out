smash> smash: got ctrl-Z
smash: process 28245 was stopped
smash> smash: got ctrl-Z
smash: process 28252 was stopped
smash> smash: got ctrl-Z
smash: process 28274 was stopped
smash> smash: got ctrl-Z
smash: process 28299 was stopped
smash> smash: got ctrl-Z
smash: process 28335 was stopped
smash> [1] sleep 100 : 28245 0 secs (stopped)
[2] sleep 200 : 28252 0 secs (stopped)
[3] sleep 300 : 28274 0 secs (stopped)
[4] sleep 400 : 28299 0 secs (stopped)
[5] sleep 500 : 28335 0 secs (stopped)
smash> smash> smash> smash> sleep 500 : 28335
smash> sleep 400 : 28299
smash> [1] sleep 100 : 28245 0 secs (stopped)
[2] sleep 200 : 28252 0 secs (stopped)
[3] sleep 300 : 28274 0 secs (stopped)
[4] sleep 400 : 28299 0 secs
[5] sleep 500 : 28335 0 secs
smash> smash: sending SIGKILL signal to 5 jobs:
28245: sleep 100
28252: sleep 200
28274: sleep 300
28299: sleep 400
28335: sleep 500
